src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAWebApp.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.PlatformAbstractions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class GivenThatWeWantToPublishAWebApp : SdkTest
+    {
+        public GivenThatWeWantToPublishAWebApp(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
+        public void It_publishes_as_framework_dependent_by_default()
+        {
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("WebApp")
+                .WithSource();
+
+            var args = new[]
+            {
+                "-p:Configuration=Release"
+            };
+
+            var restoreCommand = new RestoreCommand(Log, testAsset.TestRoot);
+            restoreCommand
+                .Execute(args)
+                .Should()
+                .Pass();
+
+            var command = new PublishCommand(Log, testAsset.TestRoot);
+
+            command
+                .Execute(args)
+                .Should()
+                .Pass();
+
+            var publishDirectory =
+                command.GetOutputDirectory(targetFramework: "netcoreapp2.0", configuration: "Release");
+
+            publishDirectory.Should().NotHaveSubDirectories();
+            publishDirectory.Should().HaveFiles(new[] {
```

Comment:
Is this the full list of files expected?  If so, then probably `OnlyHaveFiles` is better than `HaveFiles`.  If not, it might be better to list all the files anyway and use `OnlyHaveFiles` to ensure we're not getting unexpected output.

Reply:
It should be the full list.  I'll fix.

Reply:
Updated.